scipy/spatial/ckdtree.pyi

```diff
@@ -1 +1,17 @@
-class cKDTree: ...
+from typing import Union, Optional
+import numpy as np
+
+class cKDTree:
+    def __init__(self,
+                 data: np.ndarray,
+                 leafsize: int = 16,
```

Comment:
In stubs like this it's conventional to leave the actual default values out, so this would become `leafsize: int = ...`. That prevents you from having to keep them in-sync with the real code.

Reply:
Ok, I've made this change for arguments in the header/stub file.